Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/TitleOverlayLabel.h

```diff
@@ -0,0 +1,11 @@
+//  Created by Monte Hurd on 7/30/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <UIKit/UIKit.h>
+#import "PaddedLabel.h"
+
+@interface TitleOverlayLabel : PaddedLabel
```

Comment:
Said on hangout, but just adding here as well:
Lets put this in a nib and add the label as a subview. Also use a standard UILabel for the actual text (can configure in the nib).